### PR TITLE
Mw/dev

### PR DESCRIPTION
**Pull request by michaelwood**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,7 +82,11 @@ There is a webserver that provides data as API's.
 
 To run this:
 
-`$ node ./src/bin/web-server.js`
+`$ npm run start-webserver`
+
+or start in developer mode (auto reloads on changes)
```
Comment (odscjames) [inline README.md:87]: Nice!

**Comment by michaelwood** [comment]
> What's here looks great but shouldn't there be changes to package-lock.json too?

I'm not sure whether we put dev dependencies in there too?

**Comment by odscjames** [comment]
Mocha is already in there, and it's a dev dep so I assumed npm did.

**Comment by odscjames** [comment]
Yeah,  `"mocha": {` is in with `"dev": true,`. As far as I'm concerned, if npm puts it in I'm basically following that, I'm not going to second guess it.

**Comment by michaelwood** [comment]
> Yeah, `"mocha": {` is in with `"dev": true,`. As far as I'm concerned, if npm puts it in I'm basically following that, I'm not going to second guess it.

OK. I've added update `package-lock` as a separate commit for ease of reviewing. To update it I removed my `node_modules` dir and re-ran `npm install` to make sure there wasn't any other junk in there that might be brought in.

**Comment by odscjames** [comment]
Oh, I was seeing oddness here and I've worked out the root cause: this pull request adds nodemon to both dev AND non-dev dependencies. Is this intentional?

At this stage can I suggest we split this pull request. The actual work you need in is just the changes to src/lib/web-publisher-status.js and they look great. Let's got those in and sort the rest later.

**Comment by odscjames** [comment]
Essentially lets cherry pick 1009cec7b1e3cb3502f021d097c4faf5e82092f5

**Comment by odscjames** [comment]
https://github.com/openactive/conformance-services/pull/45

**Pull request merged**